docs/architecture/adr-037-gov-split-vote.md

```diff
@@ -35,22 +35,22 @@ type Vote struct {
 }
```

Comment:
feel free to put this ADR as `accepted`!

Reply:
Marked as `Accepted`